backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -573,6 +575,8 @@ export class UserResolver {
     }
 
     const user = getUser(context)
+    const backupOriginalUser = user
+
```

Comment:
hast du das getestet?
In JavaScript werden Objekte und Arrays per Referenz übergeben. Das bedeutet, dass `backupOriginalUser` auf dasselbe Objekt im Speicher verweist wie `user`, und Änderungen an einem der beiden Variablen das zugrunde liegende Objekt betreffen, auf das sie beide zeigen.

Reply:
Man könnte den Code nun sogar noch kürzer schreiben:
```ts
return !!(
    input.alias ||
    input.firstName ||
    input.lastName ||
    input.gmsAllowed ||
    input.gmsPublishLocation ||
    input.gmsPublishName ||
    input.language ||
    input.gmsLocation
);
```